cmd/compose/alpha.go

```diff
@@ -31,7 +31,6 @@ func alphaCommand(p *ProjectOptions, dockerCli command.Cli, backend api.Service)
 		},
 	}
 	cmd.AddCommand(
-		watchCommand(p, dockerCli, backend),
```

Comment:
shall we (temporary) keep this alpha command with a warning for user to adopt the official one ?

Reply:
Yes sure 👍